pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>zephyr-enterprise-test-management</artifactId>
-  <version>2.4-SNAPSHOT</version>
+  <version>2.5-SNAPSHOT</version>
```

Comment:
@KrzysiekAgi You can revert this version change.
The release is maintained through an automated process on the main repo and change is reflected here through pull after a release is made. I will do that update so this PR doesn't need to.